crates/forge/src/lib.rs

```diff
@@ -581,13 +581,19 @@ fn run_with_fuzzing(
         }
 
         let mut results = vec![];
+        let mut result = TestCaseSummary::Failed {
+            name: case.name.to_string(),
+            msg: None,
+            arguments: vec![],
+            fuzzing_statistic: None,
+        };
```

Comment:
Is it possible at all for this default to be returned? Because if yes, we will sometimes show quite weird outputs from the fuzzer, where the runs will be non 0, but arguments will be empty. Perhaps `Skipped` is a safer default?

Reply:
I have no idea, how to trigger a situation where we will return default. IMO it will be weird if someone will create a fuzzing test without subtests 🤔  I assume that if this happened FAIL is a more accurate message than skip. 

IMO we should panic in this case, but no idea how to do that